core/unit_test/TestAtomicOperations_shared.hpp

```diff
@@ -19,7 +19,8 @@
 namespace Test {
 
 // FIXME_SYCL This doesn't work yet for SYCL+CUDA
-#if !defined(KOKKOS_ENABLE_SYCL) || defined(KOKKOS_ARCH_INTEL_GPU)
+#if !defined(KOKKOS_ENABLE_OPENMPTARGET) || !defined(KOKKOS_ENABLE_SYCL) || \
+    defined(KOKKOS_ARCH_INTEL_GPU)
```

Comment:
Do you mean to disable the test independent of the compiler for `OpenMPTarget`?

Reply:
```suggestion
// FIXME OPENMPTARGET
#if !defined(KOKKOS_ENABLE_OPENMPTARGET) || !defined(KOKKOS_ENABLE_SYCL) || \
    defined(KOKKOS_ARCH_INTEL_GPU)
```

Reply:
No you are right. I fixed the condition on when to skip this test.